obs-browser-source.cpp

```diff
@@ -352,7 +358,11 @@ void BrowserSource::Update(obs_data_t *settings)
 		    n_shutdown == shutdown_on_invisible &&
 		    n_restart == restart &&
 		    n_css == css &&
-		    n_url == url) {
+		    n_url == url &&
+		    n_javascript == javascript &&
+            n_css_active == css_active &&
```

Comment:
Style:  Use tabs to indent, and then spaces to align this and the following line.  Use the lines above as examples.